config/crd/bases/ingress-nodefw.ingress-nodefw_ingressnodefirewalls.yaml

```diff
@@ -102,11 +99,17 @@ spec:
                             - udp
                             - sctp
                             type: string
+                        required:
+                        - order
+                        - protocol
                         type: object
                       type: array
-                  required:
-                  - fromCIDRs
-                  - rules
+                    sourceCIDRs:
+                      description: FromCIDRS is A list of CIDR from which we apply
```

Comment:
```suggestion
                      description: sourceCIDRS is A list of CIDR(s) from which we apply
```

Reply:
this is auto gen once I fixed the descr in types it will get fixed here